modules/admin_manual/pages/configuration/files/encryption/encryption_configuration_quick_guide.adoc

```diff
@@ -102,6 +102,9 @@ They need to:
 
 === Decrypt Encrypted Files
 
+If you have an instance with a few users, you can use this example to decrypt the files.
+Note that you have to enter the password for each user manually.
+
```

Comment:
I would specify that when running `encryption:decrypt-all` the recovery-key option must be chosen and not the user password one.

Reply:
specified